tests/app/main/forms/test_choose_time_form.py

```diff
@@ -4,26 +4,28 @@
 from app.main.forms import ChooseTimeForm
 
 
-@freeze_time("2016-01-01 11:09:00.061258")
+@freeze_time("2016-01-01 16:09:00.061258")  # this is during DST
 def test_form_contains_next_24h(notify_admin):
 
     choices = ChooseTimeForm().scheduled_for.choices
 
     # Friday
     assert choices[0] == ('', 'Now')
-    assert choices[1] == ('2016-01-01T12:00:00', 'Today at midday')
-    assert choices[13] == ('2016-01-02T00:00:00', 'Today at midnight')
+    assert choices[1] == ('2016-01-01T17:00:00', 'Today at noon ET')
+    assert choices[13] == ('2016-01-02T05:00:00', 'Today at midnight ET')
 
     # Saturday
-    assert choices[14] == ('2016-01-02T01:00:00', 'Tomorrow at 1am')
-    assert choices[37] == ('2016-01-03T00:00:00', 'Tomorrow at midnight')
+    assert choices[14] == ('2016-01-02T06:00:00', 'Tomorrow at 1am ET')
+    assert choices[37] == ('2016-01-03T05:00:00', 'Tomorrow at midnight ET')
 
     # Sunday
-    assert choices[38] == ('2016-01-03T01:00:00', 'Sunday at 1am')
+    assert choices[38] == ('2016-01-03T06:00:00', 'Sunday at 1am ET')
 
     # Monday
-    assert choices[84] == ('2016-01-04T23:00:00', 'Monday at 11pm')
-    assert choices[85] == ('2016-01-05T00:00:00', 'Monday at midnight')
+    assert choices[62] == ('2016-01-04T06:00:00', 'Monday at 1am ET')
+    assert choices[80] == ('2016-01-05T00:00:00', 'Monday at 7pm ET')
+    # assert choices[84] == ('2016-01-05T04:00:00', 'Monday at 11pm')
+    # assert choices[85] == ('2016-01-05T05:00:00', 'Monday at midnight')
```

Comment:
[question] Why the comment out here?

Reply:
Hmmm, let's see if I can get that working correctly.

Reply:
Okay, now future sending options are calculated correctly based on local time. Now I need to make sure that they _also_ get sent to the database correctly

Reply:
I think we're all set now; take another look and let me know what you think.